Author: Tristan Seligmann <mithrandi@debian.org>
Description: Make test-shelve.t more lenient
 When running on a slower system (eg. MIPS buildd), the age of the shelf can be
 10 seconds or more, resulting in the output alignment changing and thus a test
 failure. This patch makes the spacing be matched more leniently.
Bug: http://bz.selenic.com/show_bug.cgi?id=4124

--- a/tests/test-shelve.t
+++ b/tests/test-shelve.t
@@ -77,11 +77,11 @@ the common case - no options or filename
 ensure that our shelved changes exist
 
   $ hg shelve -l
-  default-01      (*)    changes to '[mq]: second.patch' (glob)
-  default         (*)    changes to '[mq]: second.patch' (glob)
+  default-01      (*)* changes to '[mq]: second.patch' (glob)
+  default         (*)* changes to '[mq]: second.patch' (glob)
 
   $ hg shelve -l -p default
-  default         (*)    changes to '[mq]: second.patch' (glob)
+  default         (*)* changes to '[mq]: second.patch' (glob)
   
   diff --git a/a/a b/a/a
   --- a/a/a
